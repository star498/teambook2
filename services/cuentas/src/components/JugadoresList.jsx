import React from 'react';
 
const JugadoresList = (props) => {
  return (
    <div>
      {
        props.jugadores.map((p) => {
          return (
               <div class="row">
                   <div class="col-sm-6">
                     <img src="images/perfil.png"  width="80" height="80" alt="author"/>
                   </div>
                   <div class="col-sm-6">
                     <a  key={p.id}>{ p.nombre}</a>
                     <div > <strong>{ p.email }</strong></div>
                   </div>
                   
                 </div>

          )
        })
      }
    </div>
  )
};
export default JugadoresList;