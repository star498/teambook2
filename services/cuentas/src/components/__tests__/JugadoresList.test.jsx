import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
 
import JugadoresList from '../JugadoresList';
 
const jugadores = [
  {
    'nombre': 'estrella',
    'email': 'strella@gmail.com',
    'apellido': 'barrientos',
    'celular': '4356565',
    'nivel': 0,
    'fechanacimiento': '12-15-48',
    'sexo' : 1,
    'id': 1,
    'estado': true
  },
  {
    'nombre': 'estrella',
    'email': 'strella@gmail.com',
    'apellido': 'barrientos',
    'celular': '4356565',
    'nivel': 0,
    'fechanacimiento': '12-15-48',
    'sexo' : 1,
    'id': 2,
    'estado': true
  },
];
 
test('JugadoresList renders properly', () => {
  const wrapper = shallow(<JugadoresList jugadores={jugadores}/>);
  const element = wrapper.find('h4');
  expect(element.length).toBe(2);
  expect(element.get(0).props.children).toBe('estrella');
});
test('JugadoresList renders a snapshot properly', () => {
  const tree = renderer.create(<JugadoresList jugadores={jugadores}/>).toJSON();
  expect(tree).toMatchSnapshot();
});


 
