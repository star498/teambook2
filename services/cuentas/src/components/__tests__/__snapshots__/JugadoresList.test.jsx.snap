// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JugadoresList renders a snapshot properly 1`] = `
<div>
  <div
    class="row"
  >
    <div
      class="col-sm-6"
    >
      <img
        alt="author"
        height="80"
        src="images/perfil.png"
        width="80"
      />
    </div>
    <div
      class="col-sm-6"
    >
      <a
        class="h6 author-name"
      >
        estrella
        barrientos
      </a>
      <div
        class="birthday-date"
      >
         
        <strong>
          strella@gmail.com
        </strong>
      </div>
    </div>
  </div>
  <div
    class="row"
  >
    <div
      class="col-sm-6"
    >
      <img
        alt="author"
        height="80"
        src="images/perfil.png"
        width="80"
      />
    </div>
    <div
      class="col-sm-6"
    >
      <a
        class="h6 author-name"
      >
        estrella
        barrientos
      </a>
      <div
        class="birthday-date"
      >
         
        <strong>
          strella@gmail.com
        </strong>
      </div>
    </div>
  </div>
</div>
`;
